### PR TITLE
Feature/animation on mount

### PR DESCRIPTION
**Pull request by wilsoncooked**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -45,7 +53,7 @@ export default class Header extends Component {
                 <NavLink
                   to='/clients'
                   className='nav-link'
-                  activeClassName='selected'>
+                  activeClassName={`selected grow ${didMount && 'visible'}`}>
```
Comment (laura-mso) [inline src/components/Header.js:71]: wow this is so interesting and cool!

**Review by laura-mso — COMMENTED** [review]
1. if i click on project details, the logo is not loading
2. if i click inside project details on back button i am directed to clients

probably not related to this branch and the code here

i really love the animations!!!!!

**Comment by laura-mso** [comment]
i fixed "if i click inside project details on back button i am directed to clients" in another branch, see pull requests

**Review by laura-mso — DISMISSED** [review]
❤️

**Review by aliashkevich — APPROVED** [review]

**Pull request merged**